### PR TITLE
Linolium floors in Vet Clinic

### PR DESCRIPTION
**Pull request by pisskop**
I cant believe because 1 space was deleted there were merge conflicts > :[

### CODE REVIEW

**Comment by Mshock777** [comment]
Gummy vitamins are obsolete.

**Comment by Mshock777** [comment]
Okay, nevermind, Rivet is going to restore them #16479, but I still think it wasn't a good change.

**Comment by pisskop** [comment]
It was what they were originally.  A lot of trdl but this is an addition frombefore multivitamins.

File: data/json/mapgen/veterinarian.json [hunk 1]
```diff
@@ -144,16 +144,14 @@
                 " ": "t_pavement",
                 ",": "t_pavement_y",
                 "~": "t_sidewalk",
-                "C": "t_linoleum_white",
-                "R": "t_linoleum_white",
-                "s": "t_linoleum_white",
-                "t": "t_linoleum_white",
                 "l": "t_linoleum_white",
-                "D": "t_sidewalk"
```
Comment (Mshock777) [inline data/json/mapgen/veterinarian.json]: Btw, why did you remove this line?
